iot/api-client/beta-features/commands/README.md

```diff
@@ -0,0 +1,31 @@
+# Cloud IoT Core Python Samples
+This folder contains Python samples that demonstrate an overview of the
+commands beta feature.
+
+## Quickstart
+1. Install the gCloud CLI as described in [the device manager guide](https://cloud.google.com/iot/docs/device_manager_guide).
```

Comment:
Nit, it's the "Cloud SDK" or "`gcloud` command-line tool" from my talks with editors.